solutions/agents/variables.tf

```diff
@@ -0,0 +1,50 @@
+########################################################################################################################
+# Common variables
+########################################################################################################################
+
+variable "ibmcloud_api_key" {
+  type        = string
+  description = "The API Key to use for IBM Cloud."
+  sensitive   = true
+}
+
+########################################################################################################################
+# SCC Workload Protection Agent variables
+########################################################################################################################
+
```

Comment:
I don't think you need to start all of the variables with `scc_workload_protection_agent` since thats all that this DA flavor actually provisions